<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Intermission</class>
 <widget class="QWidget" name="Intermission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="roundTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>25</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="countdownLabel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>320</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammoLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>200</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammoPicture">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>187</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammoQuantityLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>205</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="getReadyLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="collisionsLabel">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>200</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextRound">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(82, 83, 84);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NEXT ROUND</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
